schema.sql

```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
+CREATE DATABASE vet_clinic;
+
 CREATE TABLE animals (
-    name varchar(100)
-);
+    id INT,
```

Comment:
- [ ] <b>OPTIONAL:</b>   It would be nice to add `NOT NULL` and `PRIMARY KEY` constraints on ID just to make sure that it can't accept null values and it is primary key.

Reply:
Duly noted.